beets/ui/__init__.py

```diff
@@ -505,6 +505,13 @@ def human_seconds_short(interval):
     "cyan": 6,
     "white": 7
 }
+MORE_COLORS = { }
+# Generate color index mappings for 256 colors since there's no good way
+# to create meaningful names for them.
+for index in range(0, 256):
+    index = str(index)
+    MORE_COLORS[index] = index
```

Comment:
To avoid the need to create this mapping, could we instead just check whether the color given is a number? For example:

```py
try:
    index = int(color)
except ValueError:
    return
escape = ...  # something involving `index`
```

Reply:
Sure thing, however, that would make it harder to construct 'raw' strings for attributes, if we were to go down that route. Still don't know how to implement that though.

Reply:
Indeed… it seems like that would need some creative parsing logic.